docs/tutorials/tabular_prediction/tabular-kaggle.md

```diff
@@ -0,0 +1,114 @@
+                     eval_metric=eval_metric, hyperparameter_tune=False, verbosity=3,
+                     num_bagging_folds=10, stack_ensemble_levels=2) # delete last 2 arguments to reduce runtime
+
+results = predictor.fit_summary()
+```
+
+Now, we use the trained AutoGluon Predictor to make predictions on the competition's test data. It is imperative that multiple test data files are joined together in the exact same manner as the training data. Because this competition is evaluated based on the AUC (Area under the ROC curve) metric, we ask AutoGluon for predicted class-probabilities rather than class predictions (in general, when to use `predict` vs `predict_proba` will depend on the particular competition).
+
+```
+test_identity = pd.read_csv(directory+'test_identity.csv')
+test_transaction = pd.read_csv(directory+'test_transaction.csv')
+test = pd.merge(test_transaction, test_identity, on='TransactionID', how='left') # same join applied to training files
+test_data = task.Dataset(df = test) # convert to AutoGluon dataset
+del test_identity, test_transaction, test # free unused memory
+
+y_predproba = predictor.predict_proba(test_data)
+print(y_predproba[:5]) # some example predicted fraud-probabilities
+```
+
+Now that we have made a prediction for each row in the test dataset, we can submit these predictions to Kaggle. Most Kaggle competitions provide a sample submission file, in which you can simply overwrite the sample predictions with your own as we do below:
+
+```
+submission = pd.read_csv(directory+'sample_submission.csv')
+submission['isFraud'] = y_predproba
+submission.head()
+submission.to_csv(directory+'my_submission.csv', index=False)
+```
+
+We have now completed steps (4)-(6) from the top of this tutorial. To submit your predictions to Kaggle, you can run the following command in your terminal (from the appropriate directory):
+
+`kaggle competitions submit -c ieee-fraud-detection -f sample_submission.csv -m "my first submission"`
+
+You can now play with different `fit()` arguments and feature-engineering techniques to try and maximize the rank of your submissions in the Kaggle Leaderboard!
+
+
+**Tips to maximize predictive performance:**
+
+   - Be sure to specify the appropriate evaluation metric if one is specified on the competition website! If you are unsure which metric is best, then simply do not specify this argument when invoking `fit()`; AutoGluon should still produce high-quality models by automatically inferring which metric to use.
+
+   - If the training examples are time-based and the competition test examples come from future data, we recommend you reserve the most recently-collected training examples as a separate validation dataset passed to `fit()`. Otherwise, you do not need to specify a validation set yourself and AutoGluon will automatically partition the competition training data into its own training/validation sets.
```

Comment:
Mention that they should avoid bagging and stacking if this is the case, since we will use the val data in bagging currently.

Reply:
Done in new commit.  I guess we also needed to mention this fact in the fit() docs, so have done so as well.